### PR TITLE
portifolio version 2

### PR DESCRIPTION
**Pull request by looh16**
Portfolio: mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,44 +10,190 @@
...
-        <div class="social-media-icons">
-            <ul>
-                <li><a href="https://twitter.com/custodiolanga1"><img src="images/twitter.svg" alt="twitter"></a></li>
-                <li><a href="https://www.linkedin.com/in/custodio-serafim-2a318a23a"><img src="images/Linkedin_icon.svg" alt="Linkedin"></a></li>
-                <li><a href="default.asp"><img src="images/M.svg" alt="Social Midia"></a></li>
-                <li><a href="https://github.com/looh16/Portfolio"><img src="images/git.svg" alt="Github"></a></li>
-                <li><a href="default.asp"><img src="images/like.svg" alt="Social Midia"></a></li>
-              </ul> 
+        <div>
+            <section class="contact">
```
Comment (gtekle) [inline index.html:163]: - [ ] The contact section is not part of this milestone. Therefore, kindly move this code into its own PR for upcoming milestones.

File: css/styles.css [hunk 2]
```diff
@@ -94,7 +102,285 @@ ul {
...
+  border-radius: 8px;
+  margin-bottom: 6px;
+}
+
 .social-media-icons ul li {
   margin: 4px;
   padding-top: 2px;
 }
+
+.work {
```
Comment (gtekle) [inline css/styles.css:121]: - [ ] As stated in the project requirement, you should use CSS Grid layout for the project cards container and FlexBox layout to layout individual project cards' content.
image

File: index.html [hunk 3]
```diff
@@ -10,44 +10,190 @@
...
+                            no accounts or<br>sign-ups required.
+                        </div>
+                        <div class="program-languages">
+                            <ul class="languages-list">
+                                <li>html</li>
+                                <li>css</li>
+                                <li>javascript</li>
+                            </ul>
+                        </div>
+                        <button type="button" class="button">See Project</button>
```
Comment (gtekle) [inline index.html:79]: - [ ] The look of the button in your implementation is a bit different from the design. You should use a 1px width border, padding 12px, border-radius 8px, etc to make it look like the one in the Figma design.
#### Impl
image
#### Design
image

File: index.html [hunk 4]
```diff
@@ -10,44 +10,190 @@
...
-            through some of my work and<br>
-            experience! If you like what you see and<br>
-            have a project you need
-            coded, don’t<br>
-            hestiate to contact me.
-           </p> 
+
+        <div>
+            <div class="work">
+                <div class="card-tonic">
```
Comment (gtekle) [inline index.html:59]: - [ ] You should a background `#E5E5E5;` of white for the `body` but background color of white for the project cards.
- [ ] The gap between two project cards a bit narrower in your design.
- [ ] Kindly remove the border-shadow in your design of the cards to make it look like in the design.
#### Figma
image
#### Impl
image

File: index.html [hunk 5]
```diff
@@ -10,44 +10,190 @@
...
+                        <button type="button" class="button">See Project</button>
+                    </div>
+                </div>
+            </div>
         </div>
-        <div class="social-media">
-            LET’S CONNECT
+
+        <div>
+            <section class="about-my-self">
```
Comment (gtekle) [inline index.html:110]: - [ ] You should add a gap between the works section and the about me section as in the Figma design.
#### Figma
image

#### Impl
image

File: index.html [hunk 6]
```diff
@@ -10,44 +10,190 @@
...
+                    <ul>
+                        <li><a href="https://twitter.com/custodiolanga1"><img src="images/twitter.svg"
+                                    alt="twitter"></a></li>
+                        <li><a href="https://www.linkedin.com/in/custodio-serafim-2a318a23a"><img
+                                    src="images/Linkedin_icon.svg" alt="Linkedin"></a></li>
+                        <li><a href="default.asp"><img src="images/M.svg" alt="Social Midia"></a></li>
+                        <li><a href="https://github.com/looh16/Portfolio"><img src="images/git.svg" alt="Github"></a>
+                        </li>
+                        <li><a href="default.asp"><img src="images/like.svg" alt="Social Midia"></a></li>
+                    </ul>
```
Comment (gtekle) [inline index.html:139]: - [ ] Kindly remove the background color for the headings as in the Figma design.
- [ ] Kindly align the heading text and and the corresponding icons on the left and the right respectively as in the design.
#### Figma
image
#### Impl
image

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @looh16 :wave: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there-01


## Highlights
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Professional Readme

## Required Changes ♻️
_Check the comments under the review._

- [ ] You should use a more descriptive pull request title that shows the changes that are included in this pull request. For example, you could use the title of the activity as PR title. `Portfolio: mobile version` 

- [ ] You should add a descriptive pull request summary that show a summary of the changes introduced in the pull request. Kindly refer this GIF by Microverse.
pull_request_desc

- [ ] You should use meaningful commit messages. There is only one commit message for this milestone and I could have more commits and the messages can be improved to make it more meaningful that indicates the changes in that commit.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — COMMENTED** [review]
Hi @looh16 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Your project was invalidated for the following reasons:
- You submitted a previous pull request (That of last week). Kindly create a new pull request and submit the newly created one.
- Your pull request description does not explain vividly the new features or work done in this branch. Kindly make these changes. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**